lib/main.js

```diff
@@ -80,19 +83,29 @@ function neostandard (options) {
     ...files && { files },
   }
 
-  return [
+  const tsConfig = typescriptify(jsConfigs, {
+    files: ['**/*.ts', ...filesTs || []],
+    ignores,
+    name: 'neostandard/ts',
+  })
+
+  const obj = [
     // To make it a global ignore, "ignores" has to be the lone key of the config: https://eslint.org/docs/latest/use/configure/configuration-files#globally-ignoring-files-with-ignores
     ...ignores ? [{ ignores }] : [],
     ...isNonEmpty(meta) ? [{ name: 'neostandard/meta', ...meta }] : [],
     ...jsConfigs,
     ...ts
-      ? [typescriptify(jsConfigs, {
-          files: ['**/*.ts', ...filesTs || []],
-          ignores,
-          name: 'neostandard/ts',
-        })]
+      ? [tsConfig]
+      : [],
+    ...react
+      ? reactify(react === true ? 'recommended' : react, {
+        ignores,
+        tsConfig: ts ? tsConfig : undefined,
+      })
       : [],
   ]
+
+  return obj
```

Comment:
```suggestion
```

Reply:
sorry. that was me, debugging it and forgot to revert it back to `return`.